Gemfile.lock

```diff
@@ -82,3 +119,6 @@ DEPENDENCIES
   rspec
   simplecov
   test_xml
+
+BUNDLED WITH
+   1.14.6
```

Comment:
Latest stable bundler version is 1.15.4. Do you think updating and rebundling would make sense?

Reply:
This is only relevant for further development of the Gem, not for dependent projects. So I don't deem it necessary right now.